File: Data Structures & Algorithms/isomorphic-strings/submission-1.py
class Solution:
    def isIsomorphic(self, s: str, t: str) -> bool:
        test_str = ""
        map = {}
        j = 0
        for index, char in enumerate(s):
            if char in map:
                pass
            if char not in map and t[index] in map.values():
                return False
            else:
                map[char] = t[index]

        for char in s:
            test_str += map[char]


        return test_str == t

File: Data Structures & Algorithms/rotate-array/submission-1.java
class Solution {
    public void rotate(int[] nums, int k) {
        int[] copy = new int[nums.length];

        for (int i = 0; i < nums.length; i++) {
            int index = i + k;
            while (index > nums.length - 1) {
                index -= nums.length;
            }
            copy[index] = nums[i];
        }
        for (int i = 0; i < nums.length; i++) {
            nums[i] = copy[i];
        }
    }
}